sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -80,6 +80,8 @@ object ScalaNativePluginInternal {
       crossTarget.value / (moduleName.value + "-out")
     },
     nativeWorkdir := {
+      // all build tools should have the workdir
+      // ending in `native` for users' consistency
```

Comment:
I don't understand this.

Reply:
I put that in there as a suggestion. If you use `sbt` the code goes in a `native` directory. If `mill` for example also put it in a `native` then it would be easier on the developers to find the code.

Reply:
I can just remove that or reword it somehow.